### PR TITLE
Create code of conduct

### PR DESCRIPTION
**Pull request by AnneCarpenter**
Does this look good to everyone?

### CODE REVIEW

**Comment by bethac07** [comment]
LGTM

**Comment by 0x00b1** [comment]
Thanks, @AnneCarpenter. Would you mind changing the filename to `CONDUCT` (no file extension)?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting anne@broadinstitute.org or any other project team member
```
Comment (0x00b1) [inline CODE_OF_CONDUCT.md]: I recommend using your personal e-mail address.

**Comment by AnneCarpenter** [comment]
> Thanks, @AnneCarpenter. Would you mind changing the filename to `CONDUCT` (no file extension)?

I don't mind terribly! But my understanding is that CODE_OF_CONDUCT.md is the standard for Github; at least it's what you get when you choose the default. And for a newbie, the longer form is more immediately recognizable.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (AnneCarpenter) [reply CODE_OF_CONDUCT.md]: I believe I have changed it in the right place!

**Comment by 0x00b1** [comment]
We try to adopt POSIX-like style rather than GitHub recommendations (e.g. “LICENSE”).

**Comment by 0x00b1** [comment]
(For similar reasons it’d be great if you could also add 80 column line breaks.)

**Comment by AnneCarpenter** [comment]
All set w the filename change. Don't know how to check/fix 80 column line breaks, it looked like it already is that, can you help?

**Pull request merged**